lab_3_keywords_textrank/main.py

```diff
@@ -302,14 +353,26 @@ def fill_positions(self, tokens: tuple[int, ...]) -> None:
             tokens : tuple[int, ...]
                 sequence of tokens
         """
-        pass
+        for idx, token in enumerate(tokens):
+            if tokens[idx] not in self._positions:
+                self._positions[tokens[idx]] = []
+            self._positions[tokens[idx]] += [idx + 1]
```

Comment:
is there a reason you do not want to use append?

Reply:
also, doesn't variable `token` contains the same value as `tokens[idx]`?

Reply:
once i thought i had an error bc of using append, but now i get that it wasn't the case, cause it's the same thing and i already use it in other funcs. fixed